### PR TITLE
reset query

### PR DESCRIPTION
**Pull request by vairafiq**

### CODE REVIEW

**Review by obiPlabon — COMMENTED** [review]
Why do we have to reset the query here?

**Review by HeyMehedi — DISMISSED** [review]

**Review by obiPlabon — APPROVED** [review]
Okay

File: directorist-base.php [hunk 1]
```diff
@@ -786,6 +786,7 @@ public function get_popular_listings($count = 5, $listing_id = '')
 						}
```
Comment (kowsar89) [inline directorist-base.php:787]: `wp_reset_postdata()` could go just after while loop

File: directorist-base.php [hunk 2]
```diff
@@ -825,6 +826,7 @@ public function get_popular_listings($count = 5, $listing_id = '')
...
```
Comment (kowsar89) [inline directorist-base.php:827]: `wp_reset_postdata() `could go just after while loop

File: directorist-base.php [hunk 3]
```diff
@@ -844,6 +846,7 @@ public function get_popular_listings($count = 5, $listing_id = '')
...
 
 					}
```
Comment (kowsar89) [inline directorist-base.php:848]: `wp_reset_postdata()` could go just after while loop

File: directorist-base.php [hunk 4]
```diff
@@ -879,6 +882,7 @@ public function get_popular_listings($count = 5, $listing_id = '')
...
```
Comment (kowsar89) [inline directorist-base.php:884]: `wp_reset_postdata() `could go just after while loop

File: includes/model/ListingDashboard.php [hunk 5]
```diff
@@ -421,7 +421,7 @@ public function get_announcements() {
 				'content' => $announcement->post_content,
 			];
 		}
-
+
```
Comment (kowsar89) [inline includes/model/ListingDashboard.php:424]: unnecessary space, this file shouldn't have any changes

**Review by kowsar89 — CHANGES_REQUESTED** [review]

**Review by kowsar89 — APPROVED** [review]
Looks good

**Review by obiPlabon — APPROVED** [review]
Okay

**Pull request merged**